main.go

```diff
@@ -120,29 +121,29 @@ func main() {
 	}
 
 	if err := sources.Load(); err != nil {
-		log.Fatal(err)
+		log.Event(nil, "Failed to load sources", log.Error(err))
 	}
 
 	siteModel := generateModel(sources)
-	fmt.Println("Creating files...")
+	log.Event(nil, "Creating files...")
```

Comment:
Nothing functionally wrong with this but in general I think its good practice to include as much relevant info in your log message as possible. If this blew on prod logs are your friend in working out whats going on - what files are being created here?

Reply:
Thanks for this - have updated these as per discussion